minetest/minetest
# Add help formspec for CSM commands

This PR adds graphical interface for `.help` command, the similar as used on the server (`/help`).

## How to test

* Set `enable_client_modding` to `true` in minetest.conf
* Install some client mods (optional)
* Join singleplayer or any server that allows to load client mods
* Type `.help` in the chat and test the functionality of formspec.
* `.help -t` and `.help all -t` should return output to the chat without using formspec
